src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -2,21 +2,37 @@
 	<www-page
 		id="borrower-profile"
 	>
-		<article class="tw-relative tw-bg-secondary">
+		<article class="tw-relative md:tw-bg-secondary">
```

Comment:
Nathan confirmed that this is a bug, the borrower profile should have a white background on mobile resolutions so that the location pills and the progress bar background can be seen